.github/workflows/test-metal3-helm-chart.yml

```diff
@@ -0,0 +1,111 @@
+name: Test Metal3 Helm Chart
+
+on:
+  pull_request:
+    paths:
+      - '**/assets/metal3/metal3-0.2.0.tgz'
+
+jobs:
+  test-metal3-helm-chart:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+
+    - name: Install K3s
+      run: |
+        curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="server --cluster-init --write-kubeconfig-mode=644 --disable=servicelb" K3S_TOKEN=foobar sh -
+
+    - name: Install Helm
+      run: |
+        curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+        chmod +x get_helm.sh
+        ./get_helm.sh
+
+    - name: Install CertManager
+      run: |
+        export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
+        helm repo add jetstack https://charts.jetstack.io
+        helm repo update
+        helm install cert-manager jetstack/cert-manager \
+          --namespace cert-manager \
+          --create-namespace \
+          --set installCRDs=true \
+          --version v1.11.1 
+
+    - name: Extract Metal3 Helm Chart
+      id: get-chart
+      run: |
+        mkdir helm-metal3-charts
+        tar xvzf assets/metal3/metal3-0.2.0.tgz -C helm-metal3-charts/
+
+    - name: Disable provisioning network
+      run: |
+        echo "global:" > disable_provnet.yaml
+        echo "  enable_dnsmasq: false" >> disable_provnet.yaml
+        echo "  enable_pxe_boot: false" >> disable_provnet.yaml
+        echo "  provisioningInterface: \"\"" >> disable_provnet.yaml
+        echo "  provisioningIP: \"\"" >> disable_provnet.yaml
+        echo "  enable_metal3_media_server: false" >> disable_provnet.yaml
+        
```

Comment:
IMO we should make all of these the default from 0.2.0 onwards

Reply:
Agree, and once we do this, we can just remove this step from here

Reply:
https://github.com/suse-edge/charts/pull/41